pom.xml

```diff
@@ -94,12 +94,32 @@
           <configuration>
             <showWarnings>true</showWarnings>
             <showDeprecation>true</showDeprecation>
-            <failOnWarning>true</failOnWarning>
+            <fork>true</fork>
             <compilerArgs>
+              <!-- Args related to Error Prone, see: https://github.com/google/error-prone/blob/f8e33bc460be82ab22256a7ef8b979d7a2cacaba/docs/installation.md -->
```

Comment:
I think you could probably have a shorter link than this? The one @Marcono1234 gave seems fine to me, https://errorprone.info/docs/installation#maven.